src/compas/geometry/__init__.py

```diff
@@ -444,30 +444,666 @@
 
 """
 from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-
-from ._core import *  # noqa: F401 F403
-
-from .predicates import *  # noqa: F401 F403
-from .intersections import *  # noqa: F401 F403
-from .transformations import *  # noqa: F401 F403
-
-from .primitives import *  # noqa: F401 F403
-from .shapes import *  # noqa: F401 F403
-from .collections import *  # noqa: F401 F403
-
-from .bbox import *  # noqa: F401 F403
-from .bestfit import *  # noqa: F401 F403
-from .booleans import *  # noqa: F401 F403
-from .hull import *  # noqa: F401 F403
-from .icp import *  # noqa: F401 F403
-from .interpolation import *  # noqa: F401 F403
-from .offset import *  # noqa: F401 F403
-from .pointclouds import *  # noqa: F401 F403
-from .quadmesh import *  # noqa: F401 F403
-from .triangulation import *  # noqa: F401 F403
-from .trimesh import *  # noqa: F401 F403
-
-
-__all__ = [name for name in dir() if not name.startswith('_')]
+import compas
+
+from ._core import (
```

Comment:
style question: multiple grid imports from the same package for grouping, or a single grid import with interspersed comments?

Reply:
maybe the latter